code/Terrain/MarchingSquares.cs

```diff
@@ -109,33 +85,49 @@ public Model CreateWallModel()
 			}
 		}
 
-		if ( Host.IsClient )
+		_builder.AddCollisionMesh( wallVertices.ToArray(), wallTriangles.ToArray() );
+		// CreateWallModel
+
```

Comment:
Weird spacing for this comment here.

Reply:
And in a few spots down below too.

Reply:
I left comments on the old method names there so you can see where your methods went in the merge